rust_sodium-sys/build.rs

```diff
@@ -348,57 +348,65 @@ fn main_ios() {
         .to_string();
 
     // Determine SDK directory paths
-    let sdk_dir_simulator = Path::new(&xcode_dir).join(
-        "Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk",
-    );
-    let sdk_dir_ios =
-        Path::new(&xcode_dir).join("Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk");
+    let sdk_dir_simulator = unwrap!(
+        Path::new(&xcode_dir)
+            .join(
+                "Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk",
+            )
+            .to_str()
+    ).to_string();
+    let sdk_dir_ios = unwrap!(
+        Path::new(&xcode_dir)
+            .join("Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk")
+            .to_str()
+    ).to_string();
 
     // Min versions
     let ios_simulator_version_min = "6.0.0";
     let ios_version_min = "6.0.0";
 
     // Find compiler
-    let cc_helper = gcc::Config::new();
-    let cc = unwrap!(cc_helper.get_compiler().path().to_str()).to_owned();
+    let cc_helper = gcc::Build::new();
```

Comment:
Note: The `gcc` crate was replaced by [`cc`](https://crates.io/crates/cc).

Reply:
Ah - thanks - I didn't know that!

There are a few more tweaks due to be added in upcoming PRs, so we can make that change in those.  Or if you're planning on tackling #45, then feel free to make the change then?

Reply:
Sure, I can do that, together with #45. Probably not this week anymore, but by Monday or Tuesday next week.

Reply:
Good stuff - thanks 😃